test/expected/views.out

```diff
@@ -96,3 +96,93 @@ SELECT * FROM timescaledb_information.hypertable WHERE table_owner = 'super_user
  open         | open_ht    | super_user  |              1 |          3 | 24 kB      | 48 kB      |            | 72 kB      | f
 (2 rows)
 
+---Add integer table --
+CREATE TABLE test_table_int(time bigint, junk int);
+SELECT create_hypertable('test_table_int', 'time', chunk_time_interval => 10);
+NOTICE:  adding not-null constraint to column "time"
+      create_hypertable      
+-----------------------------
+ (5,public,test_table_int,t)
+(1 row)
+
+CREATE OR REPLACE function table_int_now() returns BIGINT LANGUAGE SQL IMMUTABLE as  'SELECT 1::BIGINT';
+SELECT set_integer_now_func('test_table_int', 'table_int_now');
+ set_integer_now_func 
+----------------------
+ 
+(1 row)
+
+INSERT into test_table_int SELECT generate_series( 1, 20), 100;
+ SELECT * FROM timescaledb_information.chunks WHERE hypertable_name = 'ht1' ORDER BY chunk_name;
+ hypertable_schema | hypertable_name |     chunk_schema      |    chunk_name    | primary_dimension |  primary_dimension_type  |         range_start          |          range_end           | range_start_integer | range_end_integer | is_compressed | chunk_tablespace | data_nodes 
+-------------------+-----------------+-----------------------+------------------+-------------------+--------------------------+------------------------------+------------------------------+---------------------+-------------------+---------------+------------------+------------
+ public            | ht1             | _timescaledb_internal | _hyper_1_1_chunk | time              | timestamp with time zone | Wed Dec 29 16:00:00 1999 PST | Wed Jan 05 16:00:00 2000 PST |                     |                   | false         |                  | 
+(1 row)
```

Comment:
Vertical display might make more sense and be easier to see changes in for the future.